napari_cellseg3d/model_workers.py

```diff
@@ -275,7 +285,6 @@ def inference(self):
                 EnsureChannelFirstd(keys=["image"]),
                 # Orientationd(keys=["image"], axcodes="PLI"),
                 # anisotropic_transform,
-                SpatialPadd(keys=["image"], spatial_size=pad),
```

Comment:
This is for the InferenceWorker. In practice, using sliding overlapping windows is better in terms of memory and robustness. Thus the users don't have to use images of size 2^n, because only the windows have to adhere to this requirement.